docs/_docs/tech/steps.md

```diff
@@ -713,7 +715,7 @@ Column names must not be escaped. Strings have to be escaped with quotes.
 }
 ```
 
-#### Example
+#### Example 1
```

Comment:
Not a big fan of numerating examples, better create a `#### Examples` section and give a meaningful title to each, like
`##### Basic usage` and 
`##### Column names with white-spaces`

Reply:
Ok to name the examples but I will stick to the enumeration to remain homogeneous with the other docs pages. So I will write: `Example 1: Basic usage`
If it's something we really want to change consistently, we should do that in a dedicated PR

Reply:
Not a big deal ;)